applications/advanced-concepts/zero-shot-translation.md

```diff
@@ -0,0 +1,18 @@
+---
+parent: Building and research
+title: Zero-shot translation
+description: Machine translation without parallel training data
+---
+
+**Zero-shot machine translation** is translation without parallel data between the source and target languages.
+Zero-shot translation is desirable because it can be too costly to create training data for each language pair.
+
+Zero-shot machine translation is an active area of research.
+Zero-shot models are typically lower quality than supervised translation models.
+
+## Approaches
+
+- Bridging: If a language pair `(source, target)` is desired, this approach translates twice: `(source, bridge)`, then `(bridge, target)`. The `bridge` language is also called a pivot language. English is a common choice for the pivot because many languages have training data with English.
```

Comment:
I might pull this out, because bridging has been going on since forever, whereas 'zero-shot' really only arose in connection with neural.

Rather, I'd frame this as an alternative or precursor to zero-shot.

Reply:
Also, we need a separate article on bridging anyway.  So maybe just make the article, using this text as a stub?